### PR TITLE
Update docs to new format

### PR DESCRIPTION
**Pull request by gyfchong**
Components covered here:
- Well
- VisuallyHidden
- Tooltip
- ToggleSwitch
- TimeField
- Tile
- Tag

### CODE REVIEW

File: draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -17,32 +17,13 @@ export type TooltipProps = {
    * @deprecated
    */
   inline?: boolean
-  /**
-   * Unfortunately, the content needed to be wrapped in a div. This can sometimes
-   * break the css layout. To get around this, we allow you to specify the css
-   * display value directly. If you need to need to modify more values, feel free
-   * to use the `classNameOverride` prop, but avoid it if you can.
-   */
   display?: "block" | "inline" | "inline-block" | "flex" | "inline-flex"
```
Comment (gyfchong) [inline draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx:20]: Might be a bit heavy handed, but I think this kind of description isn't really helpful within the IDE and much better shown together with examples. I had no idea what these meant until I did the example.

Comment (gyfchong) [reply draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx:20]: Tbh a bit hard to describe the use case for this...

File: draft-packages/well/docs/Well.stickersheet.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { StickerSheet } from "../../../storybook/components/StickerSheet"
+import { Well } from "../index"
+
+export default {
+  title: "Stickersheets/Well",
+} satisfies Meta<typeof Well>
+
+const ExampleContent = (): JSX.Element => <>&nbsp;</>
```
Comment (HeartSquared) [inline draft-packages/well/docs/Well.stickersheet.stories.tsx:10]: Maybe this shouldn't be empty?

File: draft-packages/well/docs/Well.stories.tsx [hunk 3]
```diff
@@ -1,143 +1,159 @@
 import React from "react"
-import { StoryFn } from "@storybook/react"
-import { TextField } from "@kaizen/draft-form"
-import { Well } from "@kaizen/draft-well"
-import { Heading, Paragraph } from "@kaizen/typography"
-import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { Meta, StoryFn, StoryObj } from "@storybook/react"
+import { Heading, Paragraph } from "../../../packages/typography"
```
Comment (HeartSquared) [inline draft-packages/well/docs/Well.stories.tsx:3]: Why did this change? 👀

File: draft-packages/tooltip/docs/Tooltip.stories.tsx [hunk 4]
```diff
@@ -1,367 +1,209 @@
...
 
-export const OverflowScroll: StoryFn<typeof Tooltip> = props => (
-  <>
-    <p>
-      Default Placement is 'above'. Scroll horizontally or vertically to view
-      the Tooltip "flip" and move according to the space of the viewport.
-      Ensuring the Tooltip does not get cut off.
-    </p>
+/**
+ * Set the duration (milliseconds) of how long the tooltip should stay visible for
```
[the remark this answers is not in the document]
Comment (gyfchong) [reply draft-packages/tooltip/docs/Tooltip.stories.tsx:173]: Seems to be what it does...?

File: draft-packages/well/docs/Well.stories.tsx (hunk 3 above)
Comment (gyfchong) [reply draft-packages/well/docs/Well.stories.tsx:3]: Not a fan of introducing packages just for docs when we can reference them directly

File: draft-packages/form/docs/ToggleSwitchField.stories.tsx [hunk 5]
```diff
@@ -114,18 +169,49 @@ const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
...
+        labelText="Label"
+        toggledStatus={toggledStatus}
+        onToggle={toggle}
+        disabled
+      />
+    )}
+  </ToggleStateContainer>
+)
+
+export const FullWidth: StoryFn<typeof ToggleSwitchField> = () => (
```
[the remark this answers is not in the document]
Comment (gyfchong) [reply draft-packages/form/docs/ToggleSwitchField.stories.tsx:186]: Lol yeah

File: draft-packages/form/docs/ToggleSwitchField.stickersheet.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { StickerSheet } from "../../../storybook/components/StickerSheet"
+import { ToggledStatus } from "../../form"
+import { ToggleSwitchField } from "../index"
+
+class ToggleStateContainer extends React.Component<
```
[the remark this answers is not in the document]
Comment (gyfchong) [reply draft-packages/form/docs/ToggleSwitchField.stickersheet.stories.tsx:7]: Ah yeah, forgot to revisit this when everything was working

File: draft-packages/tooltip/docs/Tooltip.stories.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (HeartSquared) [reply draft-packages/tooltip/docs/Tooltip.stories.tsx:173]: Pretty sure it's not what the `isInitiallyVisible` prop does 😉

File: draft-packages/well/docs/Well.stories.tsx (hunk 3 above)
Comment (HeartSquared) [reply draft-packages/well/docs/Well.stories.tsx:3]: That's fair :) Although being a monorepo, I'm pretty sure the packages are all just aliases anyway. We might need to tell eslint to ignore this check for stories though.

File: draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx [hunk 7]
```diff
@@ -17,32 +17,13 @@ export type TooltipProps = {
...
-  /**
-   * This is more a "desired position". The tooltip will automatically change
-   * its position, if there's not enough room to show it in the one specified.
-   */
   position?: Position
   text: React.ReactNode
   children?: React.ReactNode
   classNameOverride?: string
   mood?: Mood
-  /**
```
[the remark this answers is not in the document]
Comment (gyfchong) [reply draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx:36]: Missed this, will give it a try

File: draft-packages/well/docs/Well.stories.tsx (hunk 3 above)
Comment (gyfchong) [reply draft-packages/well/docs/Well.stories.tsx:3]: Which check?

File: draft-packages/well/docs/Well.stickersheet.stories.tsx (hunk 2 above)
Comment (gyfchong) [reply draft-packages/well/docs/Well.stickersheet.stories.tsx:10]: Wanted to simplify the snapshots by avoiding importing any new components, but perhaps a sentence will do

File: draft-packages/tooltip/docs/Tooltip.stories.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (gyfchong) [reply draft-packages/tooltip/docs/Tooltip.stories.tsx:173]: Oh copy pasta lol

File: draft-packages/well/docs/Well.stories.tsx (hunk 3 above)
Comment (HeartSquared) [reply draft-packages/well/docs/Well.stories.tsx:3]: The something something dependencies one

**Pull request closed without merging**